control_panel_api/tests/test_permissions.py

```diff
@@ -292,7 +283,6 @@ def test_normal_user_cant_operate_on_namespaces_with_same_prefix(self):
 
 
 class ToolDeploymentPermissionsTest(APITestCase):
-
```

Comment:
Not that it matters from the Python interpreter perspective but I thought methods inside a class were surrounded by a blank line:

> Method definitions inside a class are surrounded by a single blank line.

[PEP8 on blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines)

Reply:
I auto reformatted the page and for some reason it doesn't add blank to after class def.